packages/react-dom/src/events/__tests__/SyntheticEvent-test.js

```diff
@@ -18,19 +17,23 @@ describe('SyntheticEvent', () => {
   var createEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticEvent = require('events/SyntheticEvent').default;
     React = require('react');
     ReactDOM = require('react-dom');
     ReactTestUtils = require('react-dom/test-utils');
 
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticEvent.getPooled({}, '', nativeEvent, target);
+    createEvent = (nativeEvent, callback) => {
+      var instance;
+      var container = document.createElement('div');
+      ReactDOM.render(
+        <div ref={ref => (instance = ref)} onClick={callback} />,
+        container,
+      );
+      ReactTestUtils.SimulateNative.click(instance, nativeEvent);
     };
   });
 
-  it('should normalize `target` from the nativeEvent', () => {
+  // TODO: This is not testable with the public APIs, write a test for getEventTarget
```

Comment:
Why is it not testable? Pretty sure https://github.com/facebook/react/pull/11365 tests this case.